mopidy_beets/actor.py

```diff
@@ -14,23 +12,24 @@
 
 
 class BeetsBackend(pykka.ThreadingActor, backend.Backend):
-    uri_schemes = ['beets']
+    uri_schemes = ["beets"]
 
     def __init__(self, config, audio):
         super(BeetsBackend, self).__init__()
 
-        beets_endpoint = 'http://%s:%s' % (
-            config['beets']['hostname'], config['beets']['port'])
+        beets_endpoint = "http://%s:%s" % (
+            config["beets"]["hostname"],
+            config["beets"]["port"],
+        )
 
-        self.beets_api = BeetsRemoteClient(beets_endpoint, config['proxy'])
+        self.beets_api = BeetsRemoteClient(beets_endpoint, config["proxy"])
         self.library = BeetsLibraryProvider(backend=self)
         self.playback = BeetsPlaybackProvider(audio=audio, backend=self)
         self.playlists = None
 
 
 class BeetsPlaybackProvider(backend.PlaybackProvider):
-
     def translate_uri(self, uri):
-        track_id = uri.split(';')[1]
-        logger.debug('Getting info for track %s with id %s' % (uri, track_id))
+        track_id = uri.split(";")[1]
+        logger.debug("Getting info for track %s with id %s" % (uri, track_id))
```

Comment:
I encourage taking a search-n-edit round through the code to replace `%` and `.format()` with f-strings. More compact and reads better, once you're used to it.

Reply:
Indeed I did this in a separate commit :)
(except for the lazy evaluation of logging messages)